code/datums/perks/racial.dm

```diff
@@ -72,9 +72,9 @@
 	user.reagents.add_reagent("adrenol", 5)
 	return ..()
 
-/datum/perk/gutsandglory
-	name = "Guts and Glory"
-	desc = "You pull what made your ancestors conquer the stars from your will, letting your body recover somewhat from any internal damage at the cost of becoming exhausted while it works."
+/datum/perk/iwillsurvive
```

Comment:
```suggestion
/datum/perk/gutsandglory
```
DO NOT

Reply:
Guts and glory isn't a name that fits this perk, I agree with Sieg on this one